src/config/dynamic-theme-fixes.config

```diff
@@ -10438,21 +10438,23 @@ onet.pl
 plejada.pl
 
 INVERT
-.websiteLogo
-.serviceIcon
-a.serviceLogo > img
-.sheet
-.logoImage
+.WeatherDay_tempIcon__3Uzdm
+._3ZySwSLi_pur0unnAQO2No
```

Comment:
Could this be changed to a more generic selector, which doesn't have the random characters.

Reply:
See https://github.com/darkreader/darkreader/pull/2021/files